ci-operator/config/shiftstack/bugwatcher/shiftstack-bugwatcher-main.yaml

```diff
@@ -0,0 +1,30 @@
+build_root:
+  image_stream_tag:
+    name: shellcheck
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: Containerfile
+  from: src
+  to: bugwatcher
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: pep8
+  commands: pycodestyle --verbose ./*.py
+  container:
+    from: bugwatcher
+- as: bugwatcher
```

Comment:
it is not testing anything. It's running the shiftstack-bugwatcher workflow, which is the bot that assigns untriaged bugs

Reply:
Isn't that what the periodic defined below does?

... Ahhhh! The periodic defined below is generated because of this stanza! They're not duplicates.

Reply:
exactly!